spark/src/test/scala/org/apache/spark/sql/CometTPCDSQuerySuite.scala

```diff
@@ -141,9 +139,45 @@ class CometTPCDSQuerySuite
         "q98",
         "q99")
 
-      // TODO: enable the 3 queries after fixing the issues #1358.
-      override val tpcdsQueries: Seq[String] =
-        tpcdsAllQueries.filterNot(excludedTpcdsQueries.contains)
+      val tpcdsAllQueriesV2_7_0: Seq[String] = Seq(
+        "q5a",
+        "q6",
+        "q10a",
+        "q11",
+        "q12",
+        "q14",
+        "q14a",
+        "q18a",
+        "q20",
+        "q22",
+        "q22a",
+        "q24",
+        "q27a",
+        "q34",
+        "q35",
+        "q35a",
+        "q36a",
+        "q47",
+        "q49",
+        "q51a",
+        "q57",
+        "q64",
+        "q67a",
+        "q70a",
+        // TODO: unknown failure (seems memory usage over Github action runner) in CI with q72-v2.7
+        // in https://github.com/apache/datafusion-comet/pull/613.
+        // "q72",
```

Comment:
In latest run, I saw `Error: Process completed with exit code 143.`. It seems like the memory usage is larger than the Github action runner.

Reply:
I found a few particular queries (q72, q16) seems to use more memory than others. q72 cannot be run through sort merge join config now in the CI runner due to its resource limit, but I can run it locally.